OptiFineDoc/assets/minecraft/optifine/lang/en_us.lang

```diff
@@ -53,6 +53,16 @@ of.message.loadingVisibleChunks=Loading visible chunks
 
 of.options.skinCustomisation.ofCape=OptiFine Cape...

```

Comment:
Wonder if that still exists in this case...

Reply:
Yes, it's used for the button at the bottom of the Skin Customization screen:

![2019-02-18_14 04 30](https://user-images.githubusercontent.com/17113053/52952908-4f43fa00-3386-11e9-8d7b-f656d6e1a8f7.png)